cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -427,7 +427,7 @@ public Deployment generateDeployment(Map<String, String> annotations, boolean is
     protected List<EnvVar> getEnvVars() {
         List<EnvVar> varList = new ArrayList<>();
         varList.add(buildEnvVar(ENV_VAR_KAFKA_CONNECT_CONFIGURATION, configuration.getConfiguration()));
-        varList.add(buildEnvVar(ENV_VAR_KAFKA_CONNECT_METRICS_ENABLED, String.valueOf(isMetricsEnabled)));
+        varList.add(buildEnvVar(ENV_VAR_KAFKA_CONNECT_METRICS_ENABLED, String.valueOf(isMetricsEnabled())));
```

Comment:
`isMetricsEnabled` is defined as `protected` in the base abstract class, I think it's not so useful calling the getter method here. The same in the others derived classes.